.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 notifications:
   email: false
-sudo: false
+sudo: required
```

Comment:
Afaik, yes. But now that "I'm back" I'll test to verify this =)

Docker itself requires a user in the sudoers group (hence why the tutorials I found had this option set - I presume).